frontend/src/components/gamefield/Gamefield.js

```diff
@@ -1,27 +1,41 @@
 import React, { useState, useEffect } from 'react';
-import {io} from 'socket.io-client';
+import { io } from 'socket.io-client';
 import Board from '../drawerfield/Board';
+import Leaderboard from './Leaderboard';
+import "./Gamefield.css";
 
 function GameField() {
+    const lobbyData = JSON.parse(sessionStorage.getItem("lobbyData"));
+    console.log(lobbyData);
+    const rounds = lobbyData.rounds;
+
     const [socket, setSocket] = useState(null);
     const [localUsername, setLocalUsername] = useState(sessionStorage.getItem('username'));
```

Comment:
Elvileg a username és a lobbyId nem fog változni, emiatt elég lenne egy const-ba felvenni, mint ahogy a rouunds-ot is. Nem kell hozzá setter.